valhalla/userrequests/serializers.py

```diff
@@ -15,7 +15,25 @@
 from valhalla.common.configdb import ConfigDB
 from valhalla.userrequests.duration_utils import (get_request_duration, get_total_duration_dict,
                                                   get_time_allocation_key)
-from valhalla.common.rise_set_utils import get_rise_set_intervals
+from valhalla.common.rise_set_utils import get_rise_set_intervals, get_largest_interval
+
+
+class CadenceSerializer(serializers.Serializer):
+    start = serializers.DateTimeField()
+    end = serializers.DateTimeField()
+    period = serializers.FloatField(validators=[MinValueValidator(0.02)])
+    jitter = serializers.FloatField(validators=[MinValueValidator(0.02)])
+
+    def validate_end(self, value):
+        if value < timezone.now():
```

Comment:
This is not very intuitive, but timezone.now() does actually return UTC: 
https://docs.djangoproject.com/en/1.10/ref/utils/#django.utils.timezone.now